### PR TITLE
Bump animation above advanced

### PR DESCRIPTION
**Pull request by abchatra**

### CODE REVIEW

**Comment by abchatra** [comment]
image

File: libs/game/animation.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@
 */
 //% color="#03AA74" weight=100 icon="\uf021" block="Animation"
 //% groups='["Animate", "Advanced"]'
-//% advanced=true
+//% weight = 5
```
Comment (riknoll) [inline libs/game/animation.ts:6]:
```suggestion
//% weight=5
```

**Review by riknoll — APPROVED** [review]
LGTM! Just remove spaces around the equals

**Review by jwunderl — APPROVED** [review]
Change looks good, though we'll need to get galen to check how many tutorials need to get updated & update them to remove any 'go to advanced' type wording

**Comment by abchatra** [comment]
Yes, this is for next release, so we don't forget. We will have to update docs closer to the release than now. 

I intend to merge and test in /beta and see first before changing the docs

**Pull request merged**